app/views/state_file/questions/nj_tenant_eligibility/edit.html.erb

```diff
@@ -0,0 +1,44 @@
+<%
+    title = t(".title", filing_year: @filing_year)
+    content_for :page_title, title
+%>
+
+<% content_for :card do %>
+  <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card form-card--long" } do |f| %>
+    <h1 class="h2"><%= title %></h1>
+    <p><%= t(".label") %></p>
+
+    <div class="tight-checkboxes spacing-above-0 question-with-follow-up">
+      <%= f.cfa_checkbox(:tenant_home_subject_to_property_taxes, t(".tenant_home_subject_to_property_taxes"), options: { checked_value: "yes", unchecked_value: "no" }) %>
+
+
+      <div class="question-with-follow-up__question">
+        <%= f.cfa_checkbox(:tenant_building_multi_unit, t(".tenant_building_multi_unit"), options: { checked_value: "yes", unchecked_value: "no", "data-follow-up": "#tenant_access_kitchen_bath_followup", "aria-controls": "tenant_access_kitchen_bath_followup" }) %>
+      </div>
+      <div class="question-with-follow-up__follow-up" id="tenant_access_kitchen_bath_followup">
+        <%= f.cfa_checkbox(:tenant_access_kitchen_bath, t(".tenant_access_kitchen_bath"), options: { checked_value: "yes", unchecked_value: "no" }) %>
+      </div>
+
+      <%= f.cfa_checkbox(:tenant_more_than_one_main_home_in_nj, t(".tenant_more_than_one_main_home_in_nj"), options: { checked_value: "yes", unchecked_value: "no" }) %>
+      <%= f.cfa_checkbox(:tenant_shared_rent_not_spouse, t(".tenant_shared_rent_not_spouse"), options: { checked_value: "yes", unchecked_value: "no" }) %>
+
+      <% if current_intake.filing_status == :married_filing_separately %>
+        <%= f.cfa_checkbox(:tenant_same_home_spouse, t(".tenant_same_home_spouse"), options: { checked_value: "yes", unchecked_value: "no" }) %>
+      <% end %>
+
+    </div>
+
+    <div class="reveal">
```

Comment:
[pebble] We should use the native html summary/details tags for these

Reply:
Would the honeycrisp accordion be appropriate here & have the right accessibility? https://honeycrisp.herokuapp.com/cfa/styleguide#molecules-accordion

Reply:
That example does seem to have the right markup!

Reply:
@mmazanec22 I've been looking into this and I don't think it makes sense to fix, at least not in this ticket:
- The "reveal" class is what's styled in `state-file.scss` to have the "blue-box" styling, accordion does not have the application styling. I could edit the scss but I'd want to discuss this with the larger CFA team to ensure that's how the team wants accordions to be styled across the app
- I tried just updating this html to use summary/details html markup, but it seems like that interferes with the honeycrisp javascript and it no longer opens/closes when clicked.
- The "reveal" class is being used in this same way extensively across the application and across states. I'd rather have an accessibility ticket (either for our team or the CFA team) to fix the issue broadly in all occurrences, and I'm not sure it should block merging of this story. 

What do you think?

Reply:
Noting here what we just discussed that CfA has a reveal component in honeycrisp with aria-expanded, so I assume the honeycrisp js properly changes that value and we just need to add the attr. We can also add the [optional aria-controls](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/) (like in the accordion example in honeycrisp)

Reply:
Update - will bring up with CFA and fix in future ticket